src/onboarding/Start.js

```diff
@@ -74,7 +76,13 @@ class Start extends React.Component {
     return (
       <Main style={{ opacity: screenTransitionStyles.opacity }}>
         <Content style={screenTransitionStyles}>
-          <StartContent
+          <BreakPoint to="medium">
+            <Warning>
```

Comment:
Shall we center this banner, or make it full width? /cc @jounih 

<img width="704" alt="image" src="https://user-images.githubusercontent.com/36158/51476169-f4b67e80-1d84-11e9-88bc-880782777763.png">

Reply:
Let's make it full width I think - what width is breakpoint until we stop showing it?

Reply:
`768px`

Reply:
ok - then maybe centred? on a screen that size about half of it would be blank i think